client/js/views/GameView.vue

```diff
@@ -35,6 +35,7 @@
                     :rank="card.rank"
                     :data-opponent-hand-card="`${card.rank}-${card.suit}`"
                     class="transition-all opponent-hand-card-revealed"
+                    @click="selectCard(card)"
```

Comment:
It's going to be a little tricky to allow this on the face cards since you have targeting rules on click, @itsalaidbacklife. Could combine them perhaps, but might be worth taking another look at how those various rules work to think through a refactor.